sdk/aot/azure-aot-graalvm-support/src/main/java/com/azure/aot/graalvm/support/implementation/features/AppConfigFeature.java

```diff
@@ -22,8 +22,7 @@ public String getRootPackage() {
     @Override
     public Set<String[]> getDynamicProxies() {
         return setsOf(
-                interfaces("com.azure.data.appconfiguration.ConfigurationService"),
-                interfaces("com.azure.graalvm.tests.identity.ConfigurationService")
+                interfaces("com.azure.data.appconfiguration.ConfigurationService")
```

Comment:
why do we need this?

Reply:
This class `com.azure.graalvm.tests.identity.ConfigurationService` does not exists.

Reply:
Some of these changes are addressed in https://github.com/Azure/azure-sdk-for-java/pull/28180.